### PR TITLE
Enable memcached by default

### PR DESCRIPTION
**Pull request by kajinamit**
... because memcached is globally used as caching backend.

### CODE REVIEW

File: apis/core/v1beta1/openstackcontrolplane_types.go [hunk 1]
```diff
@@ -262,11 +262,13 @@ type RabbitmqSection struct {
 // MemcachedSection defines the desired state of Memcached services
 type MemcachedSection struct {
 	// +kubebuilder:validation:Optional
-	// +kubebuilder:default=false
+	// +kubebuilder:default=true
+	// +operator-sdk:csv:customresourcedefinitions:type=spec,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:booleanSwitch"}
```
Comment (abays) [inline apis/core/v1beta1/openstackcontrolplane_types.go:266]: I'm curious as to what this annotation is doing.  I haven't seen it before.  Is it something add by a newer version of `operator-sdk`?

Comment (kajinamit) [reply apis/core/v1beta1/openstackcontrolplane_types.go:267]: IIUC these are code markers supported by operator-sdk. While the existing annotations work for crd, these work for csv. https://v0-19-x.sdk.operatorframework.io/docs/golang/legacy/references/markers/

These were previously added to the other part of specs so I'm adding the same annotations for consistency. I noticed I didn't regenerate csv properly after making the change, which is not fixed in the latest change.

If we want I can make it a separate change.

Comment (abays) [reply apis/core/v1beta1/openstackcontrolplane_types.go:267]: Cool, I was just wondering.  I think it's fine to have it in this PR.

**Comment by kajinamit** [comment]
recheck

**Comment by abays** [comment]
@kajinamit Is this how OOO behaves as well?  You have to have Memcached to be able to use certain OpenStack services?

**Comment by kajinamit** [comment]
> @kajinamit Is this how OOO behaves as well? You have to have Memcached to be able to use certain OpenStack services?

Yes. For a recent few releases all openstack services are deployed with caching enabled, which is backed by memcached.

**Review by abays — APPROVED** [review]
/lgtm

**Pull request merged**